src/Investments/PortfolioValue.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import {
+  View,
+  Text,
+  ImageBackground,
+  ActivityIndicator,
+} from 'react-native';
+import { FontAwesome } from '@expo/vector-icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUserInvestmentShares, fetchStockPrice } from '../redux/investment/actionCreator';
+import styles from './styles';
+import cardBackground from '../../assets/img/investment/card.png';
+import { round, addComma } from '../common/numbers';
+
+export default function PortfolioValue() {
+  const dispatch = useDispatch();
+  const {
+    isLoading,
+    userShares,
+    price,
+    error,
+  } = useSelector((state) => state.investments);
+
+  useEffect(() => {
+    dispatch(fetchUserInvestmentShares(1));
+  }, []);
+
+  useEffect(() => {
+    if (userShares) {
+      const symbols = userShares.stocks.map((stock) => stock.symbol);
+      dispatch(fetchStockPrice(symbols));
+    }
+  }, [userShares]);
```

Comment:
should we move these to Investment/index.jsx? these data are also required to draw the pie chart